### PR TITLE
Clean up sci and recursive schema wording in README

### PR DESCRIPTION
**Pull request by frenchy64**

### CODE REVIEW

**Comment by frenchy64** [comment]
I kept writing until I could get the build to pass, but no luck :)

File: README.md [hunk 1]
```diff
@@ -1140,7 +1140,9 @@ Any (serializable) function can be used for `:dispatch`:
 
 ## Recursive Schemas
 
-[Local Registry](#local-registry) allows an easy way to create recursive schemas. To be recursive, the schema refers to another schema of the local registry using two useful keywords `:ref` and `:schema`. First, the schema is defined in the local registry, one of the value refer to another schema with the `:ref` keyword. The top-level schema is defined in the second argument of the `:schema` definition:
+To create a recursive schema, introduce a [local registry](#local-registry) with the first argument to `:schema`, and then wrap all recursive positions in the registry with `:ref`. Now use `:schema`'s second argument to define a schema in terms of the recursive schema(s) in the registry.
```
Comment (ikitommi) [inline README.md:1143]: Thanks! The local registry key can be set to any Schema via schema property `:registry`, not just to `:schema`:

```clj
[:map {:registry {::x :int, ::y :int}} ::x ::y]
```

Comment (frenchy64) [reply README.md]: Thanks, I removed any concrete schema from the first sentence, and then pointed out that the example uses `:schema`. How's that?

Comment (ikitommi) [reply README.md]: that's good.

**Comment by ikitommi** [comment]
thanks!

**Pull request merged**